### ISSUE TITLE
Amniotic infection syndrome

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6179]
**Preferred term label:**
Amniotic infection syndrome

**Synonyms**
Amniotic infection, AIS

**Definition (free text, please give PubMed ID)**
The amniotic infection "syndrome" comprises the inflammatory lesions displayed by the placenta, cord and membranes and by the fetal organs (essentially by the lungs) which may be contaminated before and after birth.
PMID: 13801291 

**Parent term (use hpo.jax.org/app)**
Unusual infection

**Comment by pnrobinson** [comment]
I do not think this is an unusual infection in the sense that it does not necessarily point to an immune defect. We would characterize this as a disease and use a Mondo term. https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0020710
I do not see any clear distinction to Amnionitis, possibly this should be a synoynm?
@nicolevasilevsky
